os_migrate/roles/conversion_host/tasks/main.yml

```diff
@@ -117,3 +117,20 @@
     ca_cert: "{{ os_migrate_conversion_ca_cert|default(omit) }}"
```

Comment:
We need to set `auto_ip: no` above.

Reply:
Also another thing that we wanted to support here was explicit assignment of pre-created FIPs (see the card description in Jira). So we could have something like `floating_ips: "{{ os_migrate_conversion_floating_ips|default(omit) }}"` here and on the higher level we'd have the `os_migrate_src_conversion_floating_ips` and `os_migrate_dst_conversion_floating_ips` split as we have with the other variables?

We could also make this a separate card and make a separate PR for this feature if we want? So this one is optional i think but the `auto_ip` change needs to be made at least.

Reply:
both are done, I didn't add this change at first because I didn't (don't) know how to test it in our CI currently.

Reply:
another thing I didnt like is to duplicate the task